### DISCUSSION TITLE
Local shared preset file is ignored

### DISCUSSION THREAD

**Discussion by bjhartin** [renovatebot/renovate#8531] [Request Help] [answered]
Hello,

I have a local preset (renovate-config repo in same org) but renovate ignores it, looking in npm.mycompany.com instead.

I have an enterprise Github repo, `github.company.com/myorg/myrepo` with a renovate.json file of:

```json
{
"extends": [
"my-org:default"
]
}
```

I also have a shared preset repo, `github.company.com/myorg/renovate-config`.  It looks like it is trying to find it as an npm package:

```bash
DEBUG: Repository config (repository=myorg/myrepo)
"fileName": "renovate.json",
"config": {"extends": ["myorg:default"]}
...
DEBUG: Dependency lookup failure: not found (repository=myorg/myrepo)
"packageName": "renovate-config-myorg"
DEBUG: Response code 404 (Not Found) (repository=myorg/myrepo)
...
DEBUG: Preset fetch error (repository=myorg/myrepo)
[...] Cannot find preset's package (myorg:default)
```

Things I've tried:

- package.json vs renovate.json in the shared preset repo
- use of "local>myorg:default"

**Comment by bjhartin** [comment, marked as answer]
My problem was that I needed to use `myorg/renovate-config` instead of `myorg:renovate-config`.

**Reply by viceice** [reply]
Yes, it needs org and repo name (and a slash) to be a file preset.
